### PR TITLE
Add logger, two class exception and Exception filter

### PR DESCRIPTION
**Pull request by artem03**
Add logger, two class exception and Exception filter

### CODE REVIEW

File: App.Accounts/Controllers/AccountsController.cs [hunk 1]
```diff
@@ -9,31 +10,37 @@ namespace App.Accounts.Controllers
...
 		{
+			_logger = logger;
 			_accountsManager = accountsManager;
 		}
 
 		// GET api/example/values
 		[HttpGet]
 		public ActionResult<List<Account>> Get()
 		{
+			_logger.LogDebug("Call Get method");
```
Comment (artem03) [inline App.Accounts/Controllers/AccountsController.cs]: Debug -> info

File: App.Accounts/Exceptions/EntityNotFoundException.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace App.Accounts.Exceptions
+{
+    public class EntityNotFoundException : Exception
```
Comment (artem03) [inline App.Accounts/Exceptions/EntityNotFoundException.cs]: Exception => SystemException

File: App.Accounts/Repositories/InMemoryAccountsRepository.cs [hunk 3]
```diff
@@ -16,20 +17,32 @@ public class InMemoryAccountsRepositoriy : IAccountsRepository, ITransientDepend
...
 		public void UnBlockAccount(int number)
 		{
 			var item = account.Find(x => x.Number == number);
+			if (item == null)
+			{
+				throw new EntityNotFoundException(typeof(Account));
+			}
 			if (!item.IsBlocked)
 			{
 				throw new Exception("You cannot Unblock an already Unblocked account");
```
Comment (artem03) [inline App.Accounts/Repositories/InMemoryAccountsRepository.cs]: use of new exce[topn class

**Review by ghost — CHANGES_REQUESTED** [review]
ожидаю оговоренных изменений

**Review by ghost — APPROVED** [review]

**Pull request merged**